flyctl/app_config.go

```diff
@@ -182,7 +182,7 @@ func (ac *AppConfig) unmarshalNativeMap(data map[string]interface{}) error {
 			case "dockerfile":
 				b.Dockerfile = fmt.Sprint(v)
 				insection = true
-			case "build_target":
+			case "build_target", "build-target":
```

Comment:
is `build-target` needed? that seems to be new but wasn't tested so curious what else changed

Reply:
I'm not the authority on this but I just saw an inconsistency with a previous fix in #1006 and decided to fix this as well. I don't know how those two code paths are related though.